nntrainer/src/neuralnet.cpp

```diff
@@ -56,7 +56,23 @@ NeuralNetwork::NeuralNetwork() :
   iter(0),
   initialized(false),
   def_name_count(0),
-  loadedFromConfig(false) {}
+  loadedFromConfig(false),
+  is_train(true) {}
+
+NeuralNetwork::NeuralNetwork(bool istrain) :
```

Comment:
this won't be needed when the constructors are merged

Reply:
but this will make easier. I guess. otherewise we need some api for set 'is_train'

Reply:
I guess @kparichay means merging this to the default constructor with default value set. 
```suggestion
NeuralNetwork::NeuralNetwork(bool istrain = true) :
 batch_size(1),
 epochs(1),
 ...
 is_train(istrain) {}
```